ush/python/pygfs/task/aero_bmatrix.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
         _res_anl = int(self.config['CASE_ANL'][1:])
 
         _bmat_yaml = os.path.join(self.runtime_config.DATA, f"{self.runtime_config.CDUMP}.t{self.runtime_config['cyc']:02d}z.chem_diagb.yaml")
+        _diff_yaml = os.path.join(self.runtime_config.DATA, f"{self.runtime_config.CDUMP}.t{self.runtime_config['cyc']:02d}z.chem_diff.yaml")
```

Comment:
can you change all of these from diff (and DIFF) to diffusion (DIFFUSION) just to be more explicit, and not make people think it is 'difference'